mayastor-test/test_replica.js

```diff
@@ -466,7 +471,7 @@ describe('replica', function () {
           assert.equal(res.state, 'POOL_ONLINE');
           assert.equal(res.disks.length, disks.length);
           for (let i = 0; i < res.disks.length; ++i) {
-            assert.equal(res.disks[i], 'uring://' + disks[i]);
+            assert.equal(res.disks[i].includes('uring://' + disks[i]), true);
```

Comment:
why the two are not exactly equal anymore?

Reply:
It follows the same pattern elsewhere as the return value is actually `<proto>://<disk>?uuid=<uuid>`.